.github/workflows/update-go-version.yml

```diff
@@ -15,6 +15,16 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
 
+      - name: Import bot's GPG key for signing commits
+        id: import-gpg
+        uses: crazy-max/ghaction-import-gpg@01dd5d3ca463c7f10f7f4f7b4f177225ac661ee4 # v6.1.0
+        with:
+          gpg_private_key: ${{ secrets.BOT_GPG_PRIVATE_KEY }}
```

Comment:
I think it would be better to use the same variable name that we do in our other projects so that we can use the org's secrets config rather than the per-project secrets config (in the near future).

${{ secrets.BUILD_KEY }}
${{ secrets.BUILD_PASSPHRASE }}